lib/bundle/vscode_extension_installer.rb

```diff
@@ -38,7 +38,7 @@ def install(name, preinstall: true, no_upgrade: false, verbose: false, force: fa
     end
 
     def extension_installed?(name)
-      installed_extensions.include? name
+      installed_extensions.include? name.downcase
```

Comment:
```suggestion
      installed_extensions.include? name
```

similarly here

Reply:
If this is not done, then the case where the Brewfile has camelCase, whereas the system call to code returns all lower-case, then this method returns `false`. I think this line can remain due to this scenario. What do you think?

Reply:
For eg, in my Brewfile, I have "BriteSnow.vscode-toggle-quotes" (which is the exact extension Id as copied from the VSCode UI. But, the system call to `code --list-extensions` returns this as all lower case.
In this scenario, this method should return true after ignoring the case, correct?

Reply:
I have updated the test to reflect this above scenario